bin/oneoff/data_fix/time_spent_milliseconds_to_seconds

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../dashboard/config/environment'
+
+# The time_spent field in the user_levels table has been
+# recording time in milliseconds. This will cause time_spent
+# to max out at ~25 days. Instead we want to record time in
+# seconds. This will cause time_spent to max out at ~68 years
+#
+# Relevant entries include:
+# SELECT *
+# FROM user_levels
+# WHERE time_spent IS NOT NULL
+#   AND time_spent > 0;
+#
+# time_spent should be transformed as such
+# (time_spent.to_f/1000).ceil.to_i
+
+# upper limit: 2811329000;
+# lower limit:   93221000;
+
+slice = 0
+
+UserLevel.find_in_batches(start: 93_221_000, finish: 2_811_329_000, batch_size: 5000) do |user_level_slice|
+  puts "PROCESSING: slice #{slice} with starting id #{user_level_slice.first.id}."
+  ActiveRecord::Base.transaction do
```

Comment:
if UserLevels could have been deleted, do we also need the ending id of each slice? And is there anything you want to capture if it fails, like which row it failed on? (maybe this is captured in whatever error message you get by default)

Reply:
Ending id: I don't think we need it. It'll just be staring_id + 5000
Row it failed on: Hmm. I thought about this for a bit. I'm super curious if you have additional thoughts on the matter. This is my first migration, so I don't have much experience to go on. My gut feeling: It might be useful as a learning experience for me, but for the migration itself, I don't think it would end up saving any time. Since the migration logic is so short and the slices are relatively small, the time spent writing error logging would outweigh the benefits of having that logging.